.eslintrc.json

```diff
@@ -140,7 +141,7 @@
       }
     ],
     "no-proto": "error",
-    "no-redeclare": "error",
+    "no-redeclare": "off",
```

Comment:
this needs to be `error` for js and `off` for typescript. you'll need an override rule for typescript rather than turning this off globally

Reply:
If the `@typescript-eslint` version handles plain JS as well, I think we can just use that version

Reply:
The docs say the `@typescript-eslint` version extends the base `eslint/no-redeclare` rule and you should turn that one off: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-redeclare.md#rule-details

Reply:
that's wonderful, but we still use plain js in rollup repos.

Reply:
We do use the typescript parser in all rollup repos, even if they don't have TS code. This rule should work on plain JS fine, there are some [tests for it](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/tests/rules/no-redeclare.test.ts) in the eslint repo.

Reply:
I'm OK with it given the chance to validate that. I recall a while back that didn't work on plain js, but I can take another look

Reply:
I just tested and it does work on js. I added a duplicate statement to a `.js` file and got the error shown below:

```
packages/dsv/src/index.js
  13:25  error    'dsv' is already defined      @typescript-eslint/no-redeclare
```